specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -1826,7 +1826,30 @@
       "properties": {
         "relationType": {
           "description": "Type of relation. Examples include: `DosageOfMedication` or 'FrequencyOfMedication', etc.",
-          "type": "string"
+          "type": "string",
+          "x-extensible-enum":[
```

Comment:
@iscai-msft what is the best way to express an extensible enum?
I also see TA has done it differently [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json#L894-L908) by using [x-ms-enum](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-enum)

Reply:
yeah, x-ms-enum is the way to go, here @ctufts [here](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-enum) are the docs for x-ms-enum if you're interested

Reply:
To add to @iscai-msft, here are also our guildelines around extensible enums: https://github.com/microsoft/api-guidelines/blob/ed8f14661dc30ed6a35961f7eacdf04fd1e819b5/azure/Guidelines.md#use-extensible-enums.

Reply:
updated to x-ms-enum with modelAsString set to true for extensibility.